src/MultiReport/Mouse.cpp

```diff
@@ -83,9 +83,14 @@ void Mouse_::end() {
 }
 
 void Mouse_::click(uint8_t b) {
+  if (report_.buttons & b) {
+    release(b);
+    sendReport();
+  }
   press(b);
   sendReport();
   release(b);
+  sendReport();
```

Comment:
This is a change to how the HID library is "supposed" to work and I don't think I'm a fan of it. In general, things should not get sent to the host until the caller calls `sendReport` - The previous "press, sendReport, release" flow... really ought not to have done that. I *think* that a more sensible change here would be to move all this logic into sendReport.

Reply:
This whole `click()` method doesn't really match the rest of what's here at all.  It seems to have been added as a kind of "helper" function, and it probably should be handled outside this HID library entirely.  Honestly, I think the other methods are poorly named, because they suggest action, when all they really do is alter pending report data (i.e. `move()` doesn't move the cursor, it just sets some values in the report).  The changes here are just to make the existing oddball `click()` method not actually _broken_, which it was.  I think the right thing to do is remove it entirely, but that's not backwards-compatible.